subsys/zbus/zbus_iterable_sections.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/logging/log.h>
+#include <zephyr/zbus/zbus.h>
+LOG_MODULE_DECLARE(zbus, CONFIG_ZBUS_LOG_LEVEL);
+
+bool zbus_iterate_over_channels(bool (*iterator_func)(const struct zbus_channel *chan))
+{
+	STRUCT_SECTION_FOREACH(zbus_channel, chan)
+	{
```

Comment:
The bracket should go on the previous line `STRUCT_SECTION_FOREACH(zbus_channel, chan) {`

Reply:
Fixed.